### PR TITLE
Add a guide on how to share a Pixie cloud instance across multiple Pixie deployments

### PR DESCRIPTION
**Pull request by nilushancosta**
Signed-off-by: Nilushan Costa <nilushancosta@gmail.com>

https://github.com/pixie-io/docs.px.dev/issues/240

### CODE REVIEW

File: content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+## Prerequisites
+
+- A domain name that you own
+- Prerequisites of the install method your choice; [Self-hosted Pixie](/installing-pixie/install-guides/self-hosted-pixie/#prerequisites) or [Air gapped Pixie](/installing-pixie/install-guides/airgap-pixie/#prerequisites)
+
+## Steps
+
+## 1. Install the NGINX Ingress Controller
+
+Install the NGINX Ingress Controller in your Kubernetes cluster. Thereafter note the IP address assigned to the `ingress-nginx-controller` Load Balancer service. All requests to the Pixie Cloud will be sent through this.
```
Comment (aimichelle) [inline content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md]: Is there an example NGINX Ingress controller that we can link to or paste in here?

File: content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+Suppose that your Pixie custom domain name is `pixie.example.com` and the IP address obtained above is `a.b.c.d`. Two `A` records need to be created as follows.
+```
+pixie.example.com           a.b.c.d
+work.pixie.example.com      a.b.c.d
+```
+
+## 3. Install Pixie Cloud
+
+If you are using the [Self-Hosted installation](/installing-pixie/install-guides/self-hosted-pixie/),
+1. Follow steps 1 - 5 and 8 in [Deploy Pixie cloud](/installing-pixie/install-guides/self-hosted-pixie/#1.-deploy-pixie-cloud).
```
Comment (aimichelle) [inline content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md]: Small nit, let's make this `Deploy Pixie Cloud` to keep capitalization consistent.

File: content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+2. Comment lines 94 to 106 in `./scripts/create_cloud_secrets.sh`
+3. Execute the script as explained in step 9 of [Deploy Pixie cloud](/installing-pixie/install-guides/self-hosted-pixie/#1.-deploy-pixie-cloud)
```
Comment (aimichelle) [inline content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md]: Same here, let's make this `Deploy Pixie Cloud` to keep capitalization consistent.

File: content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+Two Kubernetes Ingresses are required for Pixie Cloud; One for HTTPs and the other for gRPCs communication.
+
+Create two ingresses using the following configurations
+
+1. [HTTPs]
+2. [gRPCs]
+
+```bash
+kubectl apply -f <filename>
+```
```
Comment (aimichelle) [inline content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md]: We can just make this:
```bash
kubectl apply -f k8s/cloud/overlays/exposed_services_nginx/cloud_ingress_grpcs.yaml
kubectl apply -f k8s/cloud/overlays/exposed_services_nginx/cloud_ingress_https.yaml
```

File: content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
+---
+title: "Production Readiness - Shared Cloud"
```
Comment (aimichelle) [inline content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md]: Let's just call this section "Production Readiness"

File: content/en/02-installing-pixie/index.md [hunk 6]
```diff
@@ -16,4 +16,5 @@ In this section we'll cover steps to install Pixie in specific K8s environments:
 - [Install Guides](/installing-pixie/install-guides)
     - [Community Cloud for Pixie](/installing-pixie/install-guides/community-cloud-for-pixie)
     - [Self-Hosted Pixie](/installing-pixie/install-guides/self-hosted-pixie)
+    - [Production Readiness - Shared Cloud](/installing-pixie/install-guides/production-readiness-shared-cloud)
```
Comment (aimichelle) [inline content/en/02-installing-pixie/index.md]: Can we also add a link to this page in the "Self-hosted Pixie" page? At the bottom there is the "Production Readiness (advanced) section", we should link out to this page there.

**Review by aimichelle — COMMENTED** [review]
This is great! Just a few minor comments, but overall looks great. Thanks so much for putting this together. :)

File: content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md (hunk 1 above)
Comment (nilushancosta) [reply content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md]: Pointed to the installation guide and added a screenshot as an example. Fixed through https://github.com/pixie-io/docs.px.dev/pull/245/commits/eb0693c49c18d9f219c0275b718d4f3d82dbbf5a

File: content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md (hunk 2 above)
Comment (nilushancosta) [reply content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md]: Fixed through https://github.com/pixie-io/docs.px.dev/pull/245/commits/eb0693c49c18d9f219c0275b718d4f3d82dbbf5a

File: content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md (hunk 3 above)
Comment (nilushancosta) [reply content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md]: Fixed through https://github.com/pixie-io/docs.px.dev/pull/245/commits/eb0693c49c18d9f219c0275b718d4f3d82dbbf5a

File: content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md (hunk 4 above)
Comment (nilushancosta) [reply content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md]: Fixed through https://github.com/pixie-io/docs.px.dev/pull/245/commits/eb0693c49c18d9f219c0275b718d4f3d82dbbf5a

File: content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md (hunk 5 above)
Comment (nilushancosta) [reply content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md]: Updated through https://github.com/pixie-io/docs.px.dev/pull/245/commits/eb0693c49c18d9f219c0275b718d4f3d82dbbf5a

File: content/en/02-installing-pixie/index.md (hunk 6 above)
Comment (nilushancosta) [reply content/en/02-installing-pixie/index.md]: Updated both Self-Hosted Pixie and Air-gapped installation pages and added a link to this page. Fixed through https://github.com/pixie-io/docs.px.dev/pull/245/commits/eb0693c49c18d9f219c0275b718d4f3d82dbbf5a

**Comment by nilushancosta** [comment]
Changing to a draft because I need to add an instruction for one more configuration.

**Review by aimichelle — APPROVED** [review]

**Review by vihangm — APPROVED** [review]

**Pull request merged**